plugins/action/git_publish.py

```diff
@@ -34,7 +34,7 @@
 __metaclass__ = type
 # pylint: enable=invalid-name
 
-JSONTypes = Union[bool, int, str, Dict, List]
+JSONTypes = Union[bool, int, str, Dict, List]  # type:ignore
```

Comment:
Can we please add a note explaining why we are ignoring the type checks in all places?